tests/common/include/tests/common/tools_intersection_kernel.inl

```diff
@@ -67,15 +67,14 @@ TEST(tools, intersection_kernel_single) {
     transform_store.push_back(static_context, annulus_transform);
     // The masks & their store
     mask_container_t mask_store(host_mr);
-    mask_store.template add_mask<0>(10., 10., 0);
```

Comment:
Something like this is now no longer possible

Reply:
Yes. it looks better